src/app/parcels/form/ParcelForm.tsx

```diff
@@ -142,6 +146,12 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
         }
     }, [clientDetails, clientIdForFetch]);
 
+    useEffect(() => {
+        if (editMode && !packingSlotShown) {
+            initialFormErrors.packingSlot = Errors.invalidPackingSlot;
+            setFormErrors(initialFormErrors);
+        }
+    }, []);
```

Comment:
nitpick: Let's add an empty line for separation
```suggestion
    }, []);

```